scripts/launch/demo.sh

```diff
@@ -97,15 +102,13 @@ while [[ $i -lt $panesCnt ]]; do
       stake_distr=""
   fi
 
-  pane="${window}.$i"
-
   if [[ $i -lt $n ]]; then
     tmux send-keys "$(node_cmd $i "$stats" "$stake_distr" "$wallet_args" "$system_start" "$config_dir" $exec_name) --no-ntp" C-m
   else
     # Number of transactions to send per-thread: 300
     # Concurrency (number of threads sending transactions); $CONC
     # Delay between sends on each thread: 500 milliseconds
-    tmux send-keys -t ${pane} "sleep 40s && $(bench_cmd $i "$stake_distr" "$system_start" 300 $CONC 500 neighbours)" C-m
+    tmux send-keys "sleep 40s && $(bench_cmd $i "$stake_distr" "$system_start" 3000 $CONC 500 neighbours)" C-m
```

Comment:
Should make the number of transactions per thread (3000) come from a script parameter or environment variable.